lib/jobs/general-redfish-catalog.js

```diff
@@ -94,6 +94,7 @@ function GeneralRedfishCatalogJobFactory(
                     'user': settings.username || '',
                     'pass': settings.password || '',
                 },
+                rejectUnauthorized: false,
```

Comment:
Could we use `strictSSL` instead of `rejectUnauthorized` since `rejectUnauthorized` is not documented in `request`?